metallb/run_e2e.sh

```diff
@@ -20,6 +20,12 @@ sudo firewall-cmd --zone=libvirt --permanent --add-port=179/tcp
 sudo firewall-cmd --zone=libvirt --add-port=179/tcp
 sudo firewall-cmd --zone=libvirt --permanent --add-port=180/tcp
 sudo firewall-cmd --zone=libvirt --add-port=180/tcp
+sudo firewall-cmd --zone=libvirt --permanent --add-port=3784/udp
+sudo firewall-cmd --zone=libvirt --add-port=3784/udp
+sudo firewall-cmd --zone=libvirt --permanent --add-port=3785/udp
+sudo firewall-cmd --zone=libvirt --add-port=3785/udp
+sudo firewall-cmd --zone=libvirt --permanent --add-port=4784/udp
+sudo firewall-cmd --zone=libvirt --add-port=4784/udp
```

Comment:
maybe a good comment saying 
```
3784 for BFD control packets
4784 for BFD Multihop path 
3785 for BFD Echo packets
```
so future reader of this script knows what are those numbers for ?

Reply:
Will add